hack/kube/overlays/dev-am/enduro-am.yaml

```diff
@@ -99,9 +114,9 @@ spec:
           secret:
             secretName: enduro-am-secret
             items:
-              - key: id_ed25519
+              - key: .id_ed25519.secret
```

Comment:
This keys are set by the secret generator based on the file names. I updated a single rule in the `.gitignore` file to ignore all `*.secret` files in the repo, but we could change that if we care about this keys.